plugins/@grouparoo/customerio/src/lib/client.ts

```diff
@@ -18,8 +18,8 @@ class CustomerioClient {
     return this.client.identify(customerId, payload);
   }
 
-  async trackAnonymous(data: object) {
-    return this.client.trackAnonymous(data);
+  async trackAnonymous(anonymousId: string | number, data: object) {
+    return this.client.trackAnonymous(anonymousId, data);
```

Comment:
Should we update the type of client to be more specific? I think if we changed line 5 `client: any;` to specify the type then we would have caught this ourselves.

Reply:
That's a great suggestion!  Better types added in e1cffd4b0c0ad641c3ecab758904907a02b3ae6d

<img width="803" alt="Screen Shot 2022-01-18 at 10 02 27 AM" src="https://user-images.githubusercontent.com/303226/149993222-e81363cd-3e9c-419f-a3f6-735fea69afee.png">